config.yaml

```diff
@@ -0,0 +1,26 @@
+projectName: platform
+image: jembi/platform:latest
+logPath: /tmp/logs
+
+packages:
+  - interoperability-layer-openhim
+  - reverse-proxy-nginx
+  - fhir-datastore-hapi-fhir
+  - monitoring
+  - database-postgres
+  - identity-access-manager-keycloak
+
+profiles:
+  - name: dev
+    packages:
+      - interoperability-layer-openhim
+      - database-postgres
+      - fhir-datastore-hapi-fhir
+      - monitoring
+      - identity-access-manager-keycloak
+      - database-postgres
```

Comment:
This package seems to be specified already and can be removed.

Reply:
Sorry, which one ?

Reply:
The `database-postgres` package is already defined on line 17:
https://github.com/BWHIE/hie-botswana/pull/1/files/25e2ce7b5af48b2c391dad2423891133c3ca5446#diff-d8d0422389f03d783e32e627250fe29834bd09c6361640d1ff00661dd6820034R17

Reply:
:D Oh yes 🙏